Ubuntu-clang-latest/Dockerfile

```diff
@@ -5,24 +5,23 @@ MAINTAINER Maxime Gimeno
 #install the necessary packages
 RUN apt-get update \
  && apt-get install -y python \
- && apt-get install -y subversion 
+ && apt-get install -y subversion \
+ && apt-get clean -y \
 
 #download llvm and clang from trunk
-RUN cd /tmp \
+ && cd /tmp \
  && svn co http://llvm.org/svn/llvm-project/llvm/trunk llvm \
  && cd ./llvm/tools \
- && svn co http://llvm.org/svn/llvm-project/cfe/trunk clang 
+ && svn co http://llvm.org/svn/llvm-project/cfe/trunk clang \
 
 #build llvm and clang
-RUN cd  /tmp/ \
  && mkdir build \
  && cd ./build \
  && cmake -DCMAKE_CXX_COMPILER:FILEPATH=/usr/bin/g++ ../llvm \
  && make -j"$(nproc)"
-
+ && make -j"$(nproc)" install \
 #Clean-up and set some ENV variables
-RUN cd /tmp \
- && rm -rf ./llvm
+ && cd ../ && rm -rf build llvm 
 ENV PATH="/tmp/build/bin:${PATH}"
 ENV CGAL_TEST_PLATFORM="Ubuntu-clang-latest"
 ENV CC=/tmp/build/bin/clang
```

Comment:
Have you tested?

I am surprised that the blank lines and comments in the middle of the lines '& ... \' works.
And the last line `CC=` is certainly false, now that `/tmp/build` has been removed. Check where clang has been installed. It is probably `/usr/local/bin/clang`.